bin/parse_pom_xml

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env ruby
+
+# Write a JSON string of the dependencies of a Java Maven project
+# to stdout using its pom.xml project file
+
+require 'json'
+
+dependencies_regex = %r{
+    <dependencies>
+        [\s\S]*
+    <\/dependencies>
+}x
```

Comment:
What do `%r{` and `}x` mean?   Is it a way to enclose a multiline regex?

Reply:
Yes, that's correct.